static/src/javascripts/test/spec/common/commercial/commercial-feature-policies.spec.js

```diff
@@ -158,7 +158,7 @@ describe('Commercial features', ()=> {
 
     describe('Adfree experience policy', ()=> {
         it('enabling adfree hides some commercial content', ()=> {
-            userAdPreference.hideAds = true;
+            userFeatures.isAdfree = ()=> true;
```

Comment:
@OliverJAsh - are we able to run our tests against Node 4.0 for native arrow support? I wasn't aware that our ES6 usage was contingent on JSPM being used in production code...